app/views/clients/_form.html.slim

```diff
@@ -13,10 +13,7 @@
           = f.input :last_name
       .row
         .col-xs-12.col-md-6
-          = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
-                                 end_year: Date.today.year,
-                                 include_blank: true
-
+        = render 'date_of_birth_select', f: f
```

Comment:
https://github.com/plataformatec/simple_form/wiki/Custom-inputs-examples#date-time-picker-for-twitter-bootstrap-3-using-the-datetimepicker-js-library
this?
what was used for future kids?